roles/run_hook/tasks/cmd.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Executing the instructed command.
```

Comment:
I'd be extra cautious here... Really. Allowing to inject random commands may lead to really bad situations with hacks and other things all the way - not sure we should follow that path :/.

Any specific use-case? If we can avoid that (especially since it WILL end with "nah, I want pipes and all, let's replace `command` by `shell` - and then, we end in a deadly situation)